README.md

```diff
@@ -1,44 +1,53 @@
 # Wykop Mobilny 📱
 
-> Unofficial [wykop.pl](https://wykop.pl) client for Android, written in Kotlin.
+[![Build Status][build-badge]][build]
+[![Discord][discord-badge]][discord]
+[![Support via PayPal][paypal-badge]][paypal]
 
-[![Build Status](https://travis-ci.org/feelfreelinux/WykopMobilny.svg?branch=master)](https://travis-ci.org/feelfreelinux/WykopMobilny)
-[![Discord](https://img.shields.io/discord/455024671440633857.svg)](https://discord.gg/WgQZJD3)
-[![Support via PayPal](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.me/WykopMobilny/)
+> Unofficial [wykop.pl][wykop] client for Android, written in Kotlin.
 
-_[Przeczytaj w języku Polskim](README.pl.md)_
+[Przeczytaj w języku polskim.][readme-pl]
 
 <img src="screenshots/link_details_light.png" height="33%" width="33%"><img src="screenshots/mainpage_dark.png" height="33%" width="33%"><img src="screenshots/tag_light.png" height="33%" width="33%">
 
 ## Features
 
 - Browse link feed (Main page, Upcoming, Favorite, Tags)
 - Browse entries (MyWykop, Favorite, Hot, Newest, Tags)
-- Search entries / links
-- Dark theme
+- Microblog
+- Search entries/links
+- Dark theme (gray one and for AMOLEDs)
+- Surveys
+- Private messages
+- Full customization
 
 ## Download
 
-[Download the newest release with built-in update notifier](https://github.com/feelfreelinux/WykopMobilny/releases/latest)
+You can [download the newest release with built-in update notifier][download-link] or you can get it on Google Play.
 
-## Dev
+[![Google Play Download][google-play-badge]][google-play-download]
 
-Application uses APIv2 which is currently in closed beta. APIv1 keys are not supported. In order to build this project, put `apiKey="APIKEY" apiSecret="APISECRET"` in `credentials.properties`
+## Development
 
-## Changelogs
-
-Usually I report all progress along with changelogs [here](https://wykop.pl/tag/otwartywykopmobilny).
+If you want to build your very own version of Wykop Mobilny, copy `credentials.properties.example` to `credentials.properties` and fill it with your API keys. Keep in mind, that for now, application uses API version 2 that is currently in closed beta. Version 1 keys are not supported and will not work.
```

Comment:
I might be a bit nitpicking, but APIv2 is already out to public:
https://www.wykop.pl/dla-programistow/apiv2/

Reply:
You are right, I've updated the info about it.